numpyro/contrib/render.py

```diff
@@ -111,45 +127,16 @@ def get_log_probs(sample, seed=0):
         }
 
     samples = {
-        name: site["value"]
+        name: ProvenanceArray(site["value"], frozenset({name}))
```

Comment:
@fritzo This is the place we define provenance arrays, then substitute into get_log_probs function defined above to get provenance outputs.